launcher-scripts/Linux-launcher

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+if [ ! -d "jdk8u212-b03" ] 
+then
+    tar -xzf jre.tar.gz
+fi
```

Comment:
I'd suggest we extract the JRE before building the package, so this should not be needed. But you'll need to update your task that builds the package.